lib/KModal.vue

```diff
@@ -329,7 +329,10 @@
     position: fixed;
     top: 0;
     left: 0;
-    z-index: 24;
```

Comment:
Ah - I fixed this differently here: https://github.com/learningequality/kolibri/pull/7360/files in `CoreSnackbar.index.vue` by making the z-index of that to 24 (which seemed right for Material guidelines since 24 is the highest it expects).

Kind of still hacky - but, by design, we load the snackbar component very low in the DOM - which means it shows over sibling elements and anything inside of them that has the same z-index.

Note that this change won't break anything, but if we want it one less than the snackbar, it should be 23.

Reply:
okay great - that's the way it should be. 

ref https://github.com/learningequality/kolibri/pull/6618/files#r387355934

Reply:
will revert my change